cloudbuild.yaml

```
@@ -18,6 +18,14 @@
 
 steps:
 
+#
+# Print Docker version
+#
+
+- name: "gcr.io/cloud-builders/docker"
+  id: version
```

```suggestion
  id: docker-version
```

👍🏻 hard to track otherwise.